src/libambit/libambit.rules

```diff
@@ -19,6 +19,9 @@ SUBSYSTEMS=="usb", ATTRS{idProduct}=="001a", ENV{openambit}="yes"
 # Emu (a.k.a Suunto Ambit3 Peak)
 SUBSYSTEMS=="usb", ATTRS{idProduct}=="001b", ENV{openambit}="yes"
 
+# Emu (a.k.a Suunto Ambit3 Vertical)
```

Comment:
Yes, I forgot to change it in the rule once I determined what it was. It is now corrected.